emcc.py

```diff
@@ -896,16 +899,6 @@ def parse_s_args(args):
         if '=' not in key:
           key += '=1'
 
-        # Special handling of browser version targets. A version -1 means that the specific version
-        # is not supported at all. Replace those with INT32_MAX to make it possible to compare e.g.
-        # #if MIN_FIREFOX_VERSION < 68
```

Comment:
Maybe keep this comment on the new version of this code up on line 461?

Reply:
Good catch.